src/screens/feed/styled.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+import {Button, Card, Title, Paragraph} from 'react-native-paper';
+
+export const style = {flex: 1};
```

Comment:
Let's also setup a `theme` and wire it across the component tree, a good example to do is add light/dark theme support for this app. You will need `ThemeProvider` from `styled-components`. But this can be done in the next PR.